modules/ROOT/nav.adoc

```diff
@@ -2,6 +2,7 @@
 * xref:latest-overview-concept.adoc[Concepts]
 * xref:getting-started-proxy.adoc[Getting started: Manage an API and set up a proxy for it]
 * xref:api-instance-landing-page.adoc[API Instance]
+ ** xref:configure-multiple-credential-providers.adoc[Configuring Multiple Credential Providers]
```

Comment:
@priyaayyar Can you change the main header to"Multiple Identity Providers for Client Management"?

Reply:
Done! Thanks!